Formula/mpd.rb

```diff
@@ -76,10 +70,12 @@ def install
       -Dshout=enabled
       -Dupnp=pupnp
       -Dvorbisenc=enabled
+      -Dsystemd_system_unit_dir=#{lib}/systemd/system
+      -Dsystemd_user_unit_dir=#{lib}/systemd/user
```

Comment:
From reading the `meson.build` scripts, this seems to be where they want to put this stuff.